CondFormats/GeometryObjects/src/HcalParameters.cc

```diff
@@ -0,0 +1,16 @@
+#include "CondFormats/GeometryObjects/interface/HcalParameters.h"
+
+double HcalParameters::getEtaHF(const unsigned int i) const {
+  unsigned int k = rTable.size()-i-1;
+  double eta = -log(tan(0.5*atan(rTable[k]/gparHF[4])));
+  return eta;
+}
+
+std::vector<double> HcalParameters::getEtaTableHF() const {
+
```

Comment:
@bsunanda - I suggest we update the schema to include pre-calculated eta table vector for HF.